ts/src/header-validator/validate-json.ts

```diff
@@ -250,7 +250,9 @@ function matchesPattern(
 
 function uint64(ctx: Context, j: Json): Maybe<bigint> {
   return string(ctx, j)
-    .filter((s) => matchesPattern(ctx, s, uint64Regex, 'must be a uint64'))
+    .filter((s) =>
+      matchesPattern(ctx, s, uintRegex, 'must be a non-negative integer')
```

Comment:
This PR LGTM overall.

I wonder if we should just get rid of the extra message here and instead only show the regex that must be matched, since saying it must be an integer might be interpreted as must be a JSON number.

Reply:
good idea!

Reply:
We can also change https://github.com/WICG/attribution-reporting-api/blob/d6bbe748163187f2c017a6940f5dc024dc0b6169/ts/src/header-validator/validate-json.ts#L321.

Reply:
Although, that regex actually does impose a size limit, so never mind.

Reply:
Sounds good!